packet/src/lib.rs

```diff
@@ -72,6 +73,7 @@ pub struct Meta {
     pub addr: IpAddr,
     pub port: u16,
     pub flags: PacketFlags,
+    remote_pubkey: Pubkey,
```

Comment:
Flagging that this is technically a breaking change for anyone who builds a `Meta` without using `Default` in some way, ie:

```
let m = Meta { size, addr, port, flags };
```

It's not a problem, but probably requires a breaking release

Reply:
Correct, but that does still constitute a breaking change since old code will no longer compile.